src/OSPSuite.SimModel/include/SimModel/FormulaChange.h

```diff
@@ -55,7 +55,7 @@ class FormulaChange :
 
 	//used variables are only added into the list if UseAsValue=false
 	void AppendVariablesUsedInNewFormula(std::set<int> & usedVariblesIndices, const std::set<int> & variblesIndicesUsedInSwitchAssignments);
-	void AppendUsedParameters(std::set<int> & usedParameterIDs);
+	void AppendUsedParameters(std::set<int> & usedParameterIDs, bool alwaysAppend);
```

Comment:
don't understand what this parameter means `alwaysAppend`

Reply:
Is explained here: https://github.com/Open-Systems-Pharmacology/OSPSuite.SimModel/pull/60#pullrequestreview-200233305